frontend/admin/src/components/LoginPage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth'
+import firebase from 'firebase/compat/app'
+import 'firebase/compat/auth'
+import { Login } from 'ra-ui-materialui'
+
+// Configure Firebase.
```

Comment:
This comment is useless, in the next line, the name of the const clearly indicates, that it's a firebase config.